docs/reference/contrib/sitemaps.rst

```diff
@@ -88,6 +88,7 @@ a valid, crawlable hostname. If you change the site's hostname from
     </url>
 
 
+If you change site's port to ``443``, ``https`` scheme will be used.
```

Comment:
change site's -> change the site's
``https`` scheme -> the ``https`` scheme

Reply:
thanks